src/main/java/org/broadinstitute/dsm/util/externalShipper/GBFRequestUtil.java

```diff
@@ -356,21 +356,21 @@ public void orderCancellation(ArrayList<KitRequest> kitRequests) throws Exceptio
                         String ddpParticipantId = rs.getString(DBConstants.DDP_PARTICIPANT_ID);
                         logger.info("querying ES for ddpParticipantId: "+ddpParticipantId);
```

Comment:
please remove that logging as well, otherwise we will wonder why we are not seeing any logging for the ES querying

Reply:
ok